### PR TITLE
Add support for file store configuration into Quarkus

### LINKED ISSUE #16821: Add support for file store configuration into Quarkus
There should be added something like `--storage=file --storage-file-dir=...`

Provider settings should be identical to `testsuite/utils/pom.xml`

### PR DESCRIPTION
**Pull request by vramik**
Closes #16821

### CODE REVIEW

**Comment by vramik** [comment]
Making it draft. It depends on https://github.com/keycloak/keycloak/issues/13632

File: quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/StoragePropertyMappers.java [hunk 1]
```diff
@@ -366,6 +370,21 @@ private static Optional<String> resolveMapStorageProvider(Optional<String> value
         return value;
     }
 
+    private static Optional<String> resolveMapStorageProviderSingleUseObjects(Optional<String> value, ConfigSourceInterceptorContext context) {
+        try {
+            if (value.isPresent()) {
+                return of(value.map(StorageType::valueOf)
+                        .filter(type -> ! type.equals(StorageType.file))
```
Comment (pedroigor) [inline quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/StoragePropertyMappers.java:377]:
```suggestion
                        .filter(not(StorageType.file::equals))
```

**Comment by vramik** [comment]
As https://github.com/keycloak/keycloak/pull/16001 was merged it should be ready for review now.

**Review by hmlnarik — APPROVED** [review]
Thank you for the PR!

**Pull request merged**